### ISSUE TITLE
Vue.component is not a function

### ISSUE THREAD

**Issue by connor11528** [JeffreyWay/laravel-mix#975]
I am getting an error rendering my Vue component on the screen. Everything compiles fine but when I open the browser to render the component there is an error in the console: *Vue.component is not a function*

- Laravel Mix Version: "laravel-mix": "^1.0"
- Laravel Version: "laravel/framework": "5.4.*"
- Node Version (`node -v`): v8.1.0
- NPM Version (`npm -v`): 5.0.3
- Yarn Version: v0.24.6
- OS: Mac 

### Steps To Reproduce:

```
$ laravel new test-reproduce
$ cd test-reproduce
$ composer install
$ yarn install
$ touch .env (also copied in the contents of .env.example to it)
$ php artisan key:generate
$ php artisan config:clear  
$ php artisan config:cache 
$ php artisan serve 
$ yarn run watch
```

I have this in: **welcome.blade.php**

```
<!doctype html>
<html lang="{{ app()->getLocale() }}">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>Laravel</title>

        <!-- CSRF Token -->
        <meta name="csrf-token" content="{{ csrf_token() }}">

        <!-- Fonts -->
        <link href="https://fonts.googleapis.com/css?family=Raleway:100,600" rel="stylesheet" type="text/css">

    </head>
    <body>
        <div id='app'>
            <example></example>
            
        </div>
        <script src="{{ mix('js/app.js') }}"></script>
    </body>
</html>
```

I get the error: Uncaught TypeError: Vue.component is not a function

![screen shot 2017-07-07 at 11 01 16 am](https://user-images.githubusercontent.com/3578908/27970444-b367da88-6303-11e7-9e46-9c5328bf0d18.png)

I pushed the code created in there reproduction steps that are available in a github repo here: https://github.com/connor11528/test-reproduce

**Comment by JeffreyWay** [comment]
This is fixed in 1.1.1. Had to revert this: https://github.com/JeffreyWay/laravel-mix/commit/10bb64bdced0ffd5728c9ba064a10dc86255bcfb

**Comment by connor11528** [comment]
Got it thank you. 

Simply had to run `yarn upgrade laravel-mix` (ran `yarn upgrade` before and that did not get the latest version). My bad! Thanks again
